cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -166,7 +166,9 @@ func TestAddFlagsFlag(t *testing.T) {
 			ServiceAccounts: &kubeoptions.ServiceAccountAuthenticationOptions{
 				Lookup: true,
 			},
-			TokenFile: &kubeoptions.TokenFileAuthenticationOptions{},
+			TokenFile:            &kubeoptions.TokenFileAuthenticationOptions{},
+			TokenSuccessCacheTTL: 10 * time.Second,
```

Comment:
It doesn't look like there are any command-line flags on the API server to tune these two TTL parameters. Was that an anticipated feature to come? Was their omission deliberate?

Reply:
There's already control via flags over the TTLs for "expensive" webhook token validation operations. There's no current plan to expose these as tunable settings.